sdk/firstload/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # highlight.run
 
+## 9.1.4
+
+### Patch Changes
+
+-   3e99f48ca: fix otel webjs event monkeypatch breaking mouse event listeners
```

Comment:
typescript API change is a breaking one, but this feature is not publicly released